nixpkgs_review/review.py

```diff
@@ -281,6 +281,8 @@ def parse_packages_xml(stdout: IO[bytes]) -> List[Package]:
                 description = value
             elif name == "position":
                 position = value
+        elif event == "end":
+            elem.clear()
```

Comment:
Can you explain how this fix the stackoverflow?

Reply:
The idea was that it frees resources and worked yesterday. But today I got the same error and running ``ulimit -s 16384`` fixed it. Not sure what goes on here but this may not be the right solution to it.

Reply:
Yeah. Python objects should be created on the heap not the stack. You would get the same effect by starting manual garbage collection.